### ISSUE TITLE
Fix solr core reload

### ISSUE THREAD

**Issue by sanfordd** [sciencehistory/scihist_digicoll#273]
Currently Capistrano runs 
`curl, "-s", '"localhost:8983/solr/admin/cores?action=reload&core=collection1"'`
collection1's the old name, but right now we're naming the solr core from the capistrano project as per discussion on April 3rd in Slack. 
Do we want to add a solr_core value to the local_env.yml file for Capistrano to use, or have it pull the value from the solr_url by extracting the core name?
Personally, while having the core as a different value is repetitive, it may be easier to read and handle than adding logic to strip out the core.

**Comment by jrochkind** [comment]
Yeah, if it's already being provided in local_env.yml in solr_url, we should probably just parse it out of there. Less chance of making a mistake and having them not match. 

Who wants to write the code in the app?

**Comment by sanfordd** [comment]
If we do want to keep it a distinct variable, we can just have solr_url get the core name from the variable, it'll only be defined in one place either way.

**Comment by jrochkind** [comment]
> If we do want to keep it a distinct variable, we can just have solr_url get the core name from the variable, it'll only be defined in one place either way.

In ansible, you mean?  Yeah, I guess that's fine, I dunno.

**Comment by sanfordd** [comment]
Yes in Ansible. We'll just keep the solr_url defined as is and add a new variable to local_env
solr_core: {{ capistrano_project_name }}
